libentitlement.go

```diff
@@ -11,14 +11,14 @@ import (
 )
 
 type EntitlementsManager struct {
-	profile         *secprofile.Profile
+	profile         secprofile.Profile
```

Comment:
Seems like golang best practices advise to keep non-pointer interfaces as it's equivalent to a concrete struct reference.